### PR TITLE
Compile and pack System.Runtime.Extensions as netstandard13aot

### PR DESCRIPTION
**Pull request by tarekgh**
The purpose of the change is to have aot flavor which not depending on WinRT and
to be consumed by corert.
Note that as System.Runtime.Extensions depends on System.Private.Uri, we also
did the change to have System.Private.Uri support netstandard13aot too

### CODE REVIEW

**Comment by tarekgh** [comment]
@ericstj @weshaggard  could you please have a look?

**Comment by tarekgh** [comment]
test Innerloop Ubuntu Release Build and Test please

**Comment by tarekgh** [comment]
test Innerloop Windows_NT Debug Build and Test please

**Comment by tarekgh** [comment]
hold on, I am seeing some dependencies checking errors. I'll take a look and update the changes

**Comment by tarekgh** [comment]
@ericstj @weshaggard now everything looks fine. please have a look when you have a chance. Thanks.

**Comment by tarekgh** [comment]
@ericstj @weshaggard please have a quick look

**Comment by weshaggard** [comment]
LGTM

File: src/Common/src/System/Runtime/InteropServices/Marshal.Unix.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            
+            if (length == 0)
+            {
+                return String.Empty;
+            }
+            
+            char [] chars = new char[length];
+            pBytes = (byte *) ptr;
+            
+            for (int i=0; i<length; i++)
```
Comment (jkotas) [inline src/Common/src/System/Runtime/InteropServices/Marshal.Unix.cs]: This should use UTF8 encoding instead. Adding it to the issue #6778

**Pull request merged**